control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -301,20 +397,18 @@ func buildOASContainerMain(image string, etcdHostname string, port int32, intern
 		if internalOAuthDisable {
 			c.Args = append(c.Args, "--internal-oauth-disabled=true")
 		}
-		// this list can be gathered from firewall docs: https://docs.openshift.com/container-platform/4.12/installing/install_config/configuring-firewall.html
-		defaultSampleImportContainerRegistries := "quay.io,cdn03.quay.io,cdn02.quay.io,cdn01.quay.io,cdn.quay.io,registry.redhat.io,registry.access.redhat.com,access.redhat.com,sso.redhat.com"
```

Comment:
fyi @hasueki @relyt0925 
Now removing this noproxy because the konnectivity poxy should honor the customer's proxy settings
(This was added here: https://github.com/openshift/hypershift/pull/2321)